File: 3.cpp
#include <bits/stdc++.h>

using namespace std;

/*
 * Complete the 'timeConversion' function below.
 *
 * The function is expected to return a STRING.
 * The function accepts STRING s as parameter.
 * https://www.hackerrank.com/challenges/time-conversion/problem
 */

string timeConversion(string s) {
	string output = "";
	int hrs = stoi(s.substr(0, 2));
	string remaining = s.substr(2, 6);
	char format = s[s.size() - 2];
	if(format == 'A' && hrs == 12) {
		hrs = 0;
	}
	if(format == 'P') {
		hrs += 12;
		if(hrs >= 24) {
			hrs -= 12;
		}
	}
	string hrs_str = to_string(hrs);
	if(hrs_str.size() == 1) {
		hrs_str = hrs_str.insert(0, "0");
	}
	output = hrs_str + remaining;
	return output;
}

int main()
{
	ofstream fout(getenv("OUTPUT_PATH"));

	string s;
	getline(cin, s);

	string result = timeConversion(s);

	fout << result << "\n";

	fout.close();

	return 0;
}

File: 3.py
#!/bin/python3

import math
import os
import random
import re
import sys

#
# Complete the 'breakingRecords' function below.
#
# The function is expected to return an INTEGER_ARRAY.
# The function accepts INTEGER_ARRAY scores as parameter.
# https://www.hackerrank.com/challenges/breaking-best-and-worst-records/problem
#

def breakingRecords(scores):
	maxScore = scores[0]
	minScore = scores[0]
	maxRecords = 0
	minRecords = 0
	for i in range(1, len(scores)):
		if scores[i] > maxScore:
			maxRecords += 1
			maxScore = scores[i]
		elif scores[i] < minScore:
			minRecords += 1
			minScore = scores[i]
	return [maxRecords, minRecords]

if __name__ == '__main__':
	fptr = open(os.environ['OUTPUT_PATH'], 'w')

	n = int(input().strip())

	scores = list(map(int, input().rstrip().split()))

	result = breakingRecords(scores)

	fptr.write(' '.join(map(str, result)))
	fptr.write('\n')

	fptr.close()
